src/codegeneration/ModuleTransformer.js

```diff
@@ -113,7 +110,7 @@ export class ModuleTransformer extends TempVarTransformer {
 
   wrapModule(statements) {
     var functionExpression;
-    if (transformOptions.require) {
+    if (this.options_.transformView().require) {
```

Comment:
Further down we need `parseOptions`.   If we want to cache in member properties maybe we should cache `this.options_.require` and `this.options_.transformView().destructuring ||  !this.options_.parseView().destructuring` ?